### ISSUE TITLE
Creating a checkpoint fails + saving several model files

### ISSUE THREAD

**Issue by KatieBelli** [ZFTurbo/Music-Source-Separation-Training#15]
I use MDX23C for training and did a test run of 80 epochs but after running the training script again it started at epoch 0 instead of continue training. My folders are on C:\.

It also doesn't show the loss while training on CMD.

Also, is it possible that it saves a model when the sdr is better than before?
(Example: epoch 50 had the last best sdr but now epoch 60 has a better sdr than epoch 50 etc.)
